_posts/zh_CN/posts/2016-08-23-release-0-13-0.md

```diff
@@ -62,14 +68,14 @@ excerpt: 我们很高兴地宣布比特币核心0.13.0版本正式发布了。
   几年来，比特币核心节点一直使用最低中继费用率，以帮助确定他们去会处理、中转、并在他们的内存池里存储那些未经确认的交易。每个节点都能决定自己的最低中继费率，并且如果他们收到了手续费低于该限制的交易，他们不会将其添加到自己的内存池或将其转发给他们的对等节点（虽然历史上另一种被称为<a href="https://en.bitcoin.it/wiki/Transaction_fees#Priority_transactions"> 交易优先级</a> 的机制允许一些支付低廉费用的交易被纳入到内存池并被中转）。 <br />
```

Comment:
There shouldn't be any HTML markup here, it's should be in markdown as the original.